### PR TITLE
Where in filter

### PR DESCRIPTION
**Pull request by Orest-Divintari**
It adds the ability to use whereIn filters given that scout 9.2.1 is installed.

### CODE REVIEW

File: tests/Feature/SearchTest.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 final class SearchTest extends IntegrationTestCase
 {
-    public function test_search_with_filters(): void
```
Comment (matchish) [inline tests/Feature/SearchTest.php:17]: You can test both where and wherein in one test. I prefer less acceptance tests. Low level logic should be tested in unit tests

Comment (Orest-Divintari) [reply tests/Feature/SearchTest.php]: I merged the two tests into one as requested

File: Makefile [hunk 2]
```diff
@@ -21,7 +21,6 @@ help: ## Show this help
 ---------------: ## ---------------
 
 up: ## Start all containers (in background) for development
-	sudo sysctl -w vm.max_map_count=262144
```
Comment (matchish) [inline Makefile:24]: I think we have  to leave it or check os before run
https://stackoverflow.com/questions/714100/os-detecting-makefile

**Comment by matchish** [comment]
Update https://github.com/matchish/laravel-scout-elasticsearch/blob/master/CHANGELOG.md pls and bump the version

**Comment by matchish** [comment]
`4.0.8` is ok

**Comment by Orest-Divintari** [comment]
> Update https://github.com/matchish/laravel-scout-elasticsearch/blob/master/CHANGELOG.md pls and bump the version

I updated changelog and version as requested

**Comment by matchish** [comment]
You sent PR to master. If you need it in version 5 then you need to send pr to php8 branch)

**Pull request merged**